### ISSUE TITLE
Empty 'label' replaced with 'value' when using dataprovider

### ISSUE THREAD

**Issue by ditor2** [davidstutz/bootstrap-multiselect#1214]
```
<select id="example-dataprovider"></select>
<script>
	$('#example-dataprovider').multiselect();
	var options = [
		{label: '', title: 'Option 1', value: '1', selected: true}
	];
	$('#example-dataprovider').multiselect('dataprovider', options);
</script>
```

This snippet renders this:
`<option value="1" label="1" title="Option 1" selected="selected">1</option>`

instead of this:
`<option value="1" label="" title="Option 1" selected="selected"></option>`

When label is omitted totally ( { title: 'Option 1', value: '1', selected: true} ) this behavior is ok.
But when label is explicitly set to empty string then multiselect should not override it.

**Comment by s-eckard** [comment]
I agree with the solution that the value is only used as a text if the label is missing completely.
